io.github.Icalingua.Icalingua.yml

```diff
@@ -0,0 +1,76 @@
+app-id: io.github.Icalingua.Icalingua
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+separate-locales: false
+rename-icon: icalingua
+rename-desktop-file: icalingua.desktop
+command: icalingua
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --persist=.pki
+  - --filesystem=xdg-documents:ro
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  # Build and install revolt
+  - name: icalingua
+    subdir: icalingua
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/node16/bin
+      env:
+        XDG_CACHE_HOME: /run/build/icalingua/flatpak-node/cache
+        YARN_CACHE_FOLDER: /run/build/icalingua/flatpak-node/yarn-mirror
+        npm_config_nodedir: /usr/lib/sdk/node16
+    build-commands:
+      - $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn2-setup.sh
+      - yarn install
+      - |
+        . ../flatpak-node/electron-builder-arch-args.sh
+        yarn run build $ELECTRON_BUILDER_ARCH_ARGS --config.asar=false --linux --dir
+      - cp -a build/linux*unpacked /app/icalingua
+      - sed -i 's:chat;:Chat;:g' ../pkgres/*.desktop
+      - install -Dm644 -t $FLATPAK_DEST/share/applications ../pkgres/*.desktop
+      - install -Dm644 ../pkgres/512x512.png $FLATPAK_DEST/share/icons/hicolor/512x512/apps/icalingua.png
+      - install -Dm644 ../$FLATPAK_ID.appdata.xml -t /app/share/metainfo
+      - install -Dm755 ../icalingua.sh /app/bin/icalingua
+    sources:
+      - type: git
+        url: https://github.com/Icalingua/Icalingua.git
```

Comment:
You can also download pre-compiled asar or tar.xz instead of building from source

Reply:
In flathub, server will build it rather than in user's computer, so from source may be better.

Reply:
In flathub, server will build it rather than in user's computer, so from source may be better.

Reply:
also aarch64 support.